### PR TITLE
Add test mock for google api geocoder

### LINKED ISSUE #357: Tests fail when bump LBL to 1.6.0
I noticed this the other day and @crisner said she saw the same thing when she updated npm. It is throwing errors for the PLpeopleLayer due to the changes made in LBL.

natalie@natalie-ubuntu: ~-Dev-Ruby-public-lab-leaflet-environmental-layers_022

I have a fix for this.

### PR DESCRIPTION
**Pull request by nstjean**
Fixes #357  (<=== Add issue number here)

This is the same fix as in LBLD https://github.com/publiclab/leaflet-blurred-location-display/pull/93

### CODE REVIEW

File: _SpecRunner.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>Jasmine Spec Runner</title>
-  <link rel="shortcut icon" type="image/png" href=".grunt\grunt-contrib-jasmine\jasmine_favicon.png">
```
Comment (sagarpreet-chadha) [inline _SpecRunner.html]: This file should be generated on run-time ideally, but let it be there.

File: package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "jquery": "^3.3.1",
     "leaflet": "^1.3.1",
-    "leaflet-blurred-location": "git://github.com/publiclab/leaflet-blurred-location#main",
+    "leaflet-blurred-location": "git://github.com/publiclab/leaflet-blurred-location.git#main",
     "leaflet-fullhash": "github:sagarpreet-chadha/leaflet-fullHash",
```
Comment (sagarpreet-chadha) [inline package.json]: `leaflet-fullhash` @nstjean @crisner 
We can remove this package now, right?

File: _SpecRunner.html (hunk 1 above)
Comment (nstjean) [reply _SpecRunner.html]: That's weird that it changed that line, I didn't touch that file. I don't think this file exists in LBLD, I wonder if it's actually using it at all? The `jasmine.json` file existed but was totally ignored by Jasmine.

File: package.json (hunk 2 above)
Comment (nstjean) [reply package.json]: Which one, `leaflet-fullhash`?

File: _SpecRunner.html (hunk 1 above)
Comment (nstjean) [reply _SpecRunner.html]: Yes I was right... I can delete that file and it doesn't affect the tests at all. It is being generated at runtime.

Comment (nstjean) [reply _SpecRunner.html]: Which is to say, it gets re-generated at runtime.

**Comment by nstjean** [comment]
This may no longer be needed, see my comment on https://github.com/publiclab/leaflet-blurred-location/pull/235#issuecomment-576817901

File: package.json (hunk 2 above)
Comment (crisner) [reply package.json]: Yes, It is not used in LEL anymore.

**Comment by nstjean** [comment]
This is no longer needed after https://github.com/publiclab/leaflet-blurred-location/pull/235

**Pull request closed without merging**